alluxio/worker_ring.py

```diff
@@ -205,19 +205,19 @@ class ConsistentHashProvider:
     def __init__(
         self,
         etcd_hosts=None,
-        etcd_port=2379,
+        etcd_port=None,
         worker_hosts=None,
-        worker_http_port=DEFAULT_HTTP_SERVER_PORT,
+        worker_http_port=None,
```

Comment:
I feel we can just use list of WorkerNetAddress here, What do you think

Reply:
WorkerNetAddress has too many legancy fields, most of the fields are not actually being used or are not related to current Alluxio python api

Reply:
Later refactor PR can consider how to refractor this part of code and separate the responsibilities